cypress/integration/reconciliation_spec.js

```diff
@@ -69,24 +69,25 @@ describe('Dashboard Reconciliation Reports Page', () => {
 
     it('should have a download button column', () => {
       cy.visit('/reconciliation-reports');
-      cy.get('.button__row--download').should('have.length', 3);
+      cy.get('.button__row--download').should('have.length', 4);
     });
 
     it('deletes a report when the Delete button is clicked', () => {
       cy.visit('/reconciliation-reports');
       cy.get('[data-value="inventoryReport-20200114T202529026"]').find('.button__row--delete').click({ force: true });
 
-      cy.get('.table .tbody .tr').should('have.length', 2);
+      cy.get('.table .tbody .tr').should('have.length', 3);
       cy.get('[data-value="inventoryReport-20200114T202529026"]')
         .should('not.exist');
     });
 
     it('displays a link to an individual report', () => {
```

Comment:
Do you have a test for GNF report page?

Reply:
Added one https://github.com/nasa/cumulus-dashboard/pull/840/commits/7d2937ba468683ec9d3dc6805f3910580f2c57f0